native-messaging/app/ping_pong.py

```diff
@@ -37,6 +37,7 @@ def sendMessage(encodedMessage):
         receivedMessage = getMessage()
         if receivedMessage == "ping":
             sendMessage(encodeMessage("pong3"))
+
 except AttributeError:
```

Comment:
The code here will be unreachable since `python3` is now called instead of `python` (which may be `python2` or `python3`). Let's remove the `try:` at the front and all code in the `catch` block.

Reply:
Should we just drop support for Python 2? I made use of `python3` explicit because AFAIK Python installation process no longer exposes a `python` executable and AFAIK it's not possible to declare a shebang that falls back to another scripting environment.